baggage/HTTP_HEADER_FORMAT.md

```diff
@@ -18,17 +18,17 @@ This section uses the Augmented Backus-Naur Form (ABNF) notation of [[!RFC5234]]
 
 ## Definition
 
-```
+```ABNF
 list        = list-member 0*179( OWS "," OWS list-member )
 list-member = key OWS "=" OWS value *( OWS ";" OWS property )
 property    = key OWS "=" OWS value
 property    = key OWS
```

Comment:
```suggestion
property    =/ key OWS
```

similar to https://github.com/w3c/trace-context/issues/364

Reply:
I was surprised myself but assumed it was valid syntax. I think the rule would be short enough to just write
```
property = key OWS "=" OWS value / key OWS
```
but added the incremental alternative as you suggested.